src/test/java/seedu/duke/TaskListTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void Test1(){
```

Comment:
Will the naming of test methods be better if you follow the test methods naming standards?
```suggestion
    public void addTest_addTask_expectedBehaviour( )
```